sycl/include/CL/sycl/kernel.hpp

```diff
@@ -57,6 +60,7 @@ class __SYCL_EXPORT kernel {
   ///
   /// \param ClKernel is a valid OpenCL cl_kernel instance
   /// \param SyclContext is a valid SYCL context
+  __SYCL2020_DEPRECATED("OpenCL interop APIs are deprecated")
```

Comment:
The messages sounds a little scary to those who want the OpenCL-interop, I guess. Should we say explicitly the `.get()` is deprecated and the `.get_native()` is the new way?

Reply:
Rephrased